debian/bionic/foreman/filter-package-json.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+
+import json
+
+# We need a list of packages that are devDependencies but not needed to build
```

Comment:
if you think you've seen this before, it's true, it's stolen from https://github.com/theforeman/foreman-packaging/blob/8d1663bca7d647e652ca0803772bb21586bff2e4/update-requirements#L9-L53

I wonder if this should move into `foreman.git`, at least the definitions of what to exclude.

Reply:
That actually feels like a good idea. Then it can be done in the same commit as when a dependency is added. It'll also work with reverts etc. :+1:

Reply:
The drawback is that today this list is applied to plugins too. If it lives in `foreman.git`, this wouldn't happen anymore.

Reply:
Can we ship it in `foreman-assets` then?

Reply:
Oh, I meant for `update-requirements` to use it when we package up plugins in `rpm/develop`.

Reply:
So there's 2 approaches that I can see:
* Ship a JSON file in foreman.git with the 2 lists
* Ship the entire script in foreman.git

Which one are you talking about?

Reply:
To roundup what we talked on IRC:
* put a JSON file into `foreman.git`
* rpm packaging `update-requirements` would gain a way to get that file via HTTPS from GitHub or alternatively a local checkout
* I'd probably put this script (`filter-package-json.py`) also into `foreman.git` as it allows deduplication vs having it here three times :woman_facepalming: